src/details/ArborX_DetailsHeap.hpp

```diff
@@ -114,6 +114,16 @@ KOKKOS_INLINE_FUNCTION void sortHeap(RandomIterator first, RandomIterator last,
     popHeap(first, last--, comp);
 }
 
+template <typename RandomIterator, typename Compare>
+KOKKOS_INLINE_FUNCTION void heapSort(RandomIterator first, RandomIterator last,
+                                     Compare comp)
```

Comment:
I had forgotten about `sortHeap`.  Coexistence of the two is unfortunate.
I was following the heap operations from the `std::` algorithm library when I implement these.  (I see "bubble up/down" as implementation details.)  It turns out there is a `std::make_heap` that I did not provide.  I suggest you continue following the standard and add `makeHeap` and we do `makeHeap(first, last, comp)` followed by `sortHeap(first, last, comp)` when we want to use.

Reply:
Added `makeHeap`. Still have both `sortHeap` and `heapSort`. I think it's fine.